plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/domains/NullableDomain.kt

```diff
@@ -54,6 +54,13 @@ import org.jetbrains.kotlin.formver.viper.ast.*
  *         (nx == (null_val(): Nullable[T])) ||
  *         isSubtype((typeOf(nx): Type), newType)
  *   }
+ *
+ *   axiom null_not_any {
+ *      (forall nx: Nullable[T], newType: Type ::
+ *        { !isSubtype((typeOf(nx): Type), newType) }
+ *        !is_nullable_type(newType) && nx == (null_val(): Nullable[T]) ==>
+ *          !isSubtype((typeOf(nx): Type), newType))
+ *   }
  * }
```

Comment:
I think that there's an issue with this axiom, while running the code, a lot of warnings pop out saying "Prover warning: WARNING: (1439,40): 'not' cannot be used in patterns.".

I tried to put the full dump on viper and it doesn't compile.
```
  axiom dom$Nullable$null_not_any {
    (forall nx: dom$Nullable[T], newType: dom$Type ::
      { !dom$Type$isSubtype((dom$TypeOf$typeOf(nx): dom$Type), newType) }
      !dom$Type$is_nullable_type(newType) &&
      nx == (dom$Nullable$null(): dom$Nullable[T]) ==>
      !dom$Type$isSubtype((dom$TypeOf$typeOf(nx): dom$Type), newType))
  }
```
It says Constructing the AST has failed: `{ !dom$Type$isSubtype((dom$TypeOf$typeOf(nx): dom$Type), newType) }` is not a valid Trigger